resources/js/components/events/FormField.vue

```diff
@@ -167,9 +175,21 @@ export default {
   watch: {
     events: {
       handler(newValue, oldValue) {
-        const data = _.chain(this.events).filter(event => {
-          return this.isValidEvent(event);
-        }).value();
+        const data = _.chain(this.events)
+          .filter(event => {
+            return this.isValidEvent(event);
+          })
+          .map(event => {
+            if (! this.isUpdatedEvent(event)) {
+              event.column = null;
+              event.value = null;
+              event.anyValue = true;
+            }
+
+            return event;
+          })
```

Comment:
This is the bugfix for clearing out the event data when the event "type" is not "updated"